HelpSource/scdoc.css

```diff
@@ -356,7 +357,7 @@ a.method-name {
 
 .method {
     margin-left: 2em;
-    margin-bottom: 2em;
+    margin-bottom: 3.5em;
```

Comment:
yes, sure! Does this PR still make sense? You can just use my suggestions in your redesign, if you like, and close it.